src/components/displayAll.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { FetchStockData, ShowStockData } from '../redux/Reducer';
+
+function DisplayAll() {
+  const Stocks = useSelector((state) => state.stocks);
+  const dispatch = useDispatch();
+
+  const Navigate = useNavigate();
+  useEffect(() => {
+    dispatch(FetchStockData());
+  }, []);
+
+  const obtainDetails = (symbol) => {
+    dispatch(ShowStockData(symbol));
+  };
+
+  return (
+    <>
+
+      <h3 className="main-heading"> Crypto currency market data</h3>
+
+      <div className="information">
+
+        {
+
+Stocks.length > 1
+  ? Stocks.map((stock) => (
+    <div key={stock.symbol} className="columnDiv">
+      <button
+        className=""
```

Comment:
- [ ] it would be better if you remove any spacing from the edge  in order to match the Design 
- Yours 
<img width="757" alt="Screenshot 2022-12-01 at 11 59 56" src="https://user-images.githubusercontent.com/61790587/205036608-6b4e33ba-b0e0-463d-b3b6-a44526abb1f3.png">
- Design 
<img width="205" alt="Screenshot 2022-11-30 at 17 12 55" src="https://user-images.githubusercontent.com/61790587/205036645-f3931c5c-7fc0-4e2e-a5c6-ae2e3efef1bd.png">